src/shared/lib/appNameUtils.ts

```diff
@@ -60,5 +69,28 @@ export const getAppName = (
 		case 'if_':
 		case 'ios_feast_app':
 			return 'Feast';
+		case 'ed_':
+		case 'editions_pressreader':
+			return 'Editions';
+	}
+};
+
+/**
+ * @name getNativeAppFromBowser
+ * @description Use a Bowser instance to determine if the browser is a native os.
+ * @param browser - Bowser instance
+ * @returns {IsNativeApp} - 'android' or 'ios' if the browser is a native app, otherwise undefined
```

Comment:
is this strictly true ? wouldn't this just return whether you are using a mobile device? even if you don't have the app installed this would return true if you use your phone browser, right ? Do we not care about this distinction ?

Reply:
Have updated both the name and comment to make it clearer.